script/sheet/item.js

```diff
@@ -79,6 +79,24 @@ export class ForbiddenLandsItemSheet extends ItemSheet {
         ["data.bonus.value"]: value,
       });
     });
+    html.find(".feature").click(async (ev) => {
+      const featureName = $(ev.currentTarget).data("feature");
+      const features = this.object.data.data.features;
+      if (featureName === "parrying") {
+        this.object.update({ "data.features.parrying": !features.parrying });
+      } else if (featureName === "hook") {
+        this.object.update({ "data.features.hook": !features.hook });
+      } else if (featureName === "edged") {
+        this.object.update({ "data.features.edged": !features.edged });
+      } else if (featureName === "pointed") {
+        this.object.update({ "data.features.pointed": !features.pointed });
+      } else if (featureName === "blunt") {
+        this.object.update({ "data.features.blunt": !features.blunt });
+      } else if (featureName === "slowReload") {
+        this.object.update({ "data.features.slowReload": !features.slowReload });
+      }
```

Comment:
Think this is the perfect usecase for a switch statement.
```js
switch (featureName) {
      case "parrying":
        this.object.update({ "data.features.parrying": !features.parrying });
      case "hook":
        this.object.update({ "data.features.hook": !features.hook });
      case "edged":
        this.object.update({ "data.features.edged": !features.edged });
      case "pointed":
        this.object.update({ "data.features.pointed": !features.pointed });
      case "blunt":
		this.object.update({ "data.features.blunt": !features.blunt });
      case "slowReload":
        this.object.update({ "data.features.slowReload": !features.slowReload });
      }
}
```

Reply:
… Indeed it is. 😅 

- [ ] Use a `switch` statement instead of a bazillion `else if`